ConnectorRevit/ConnectorRevit/UI/ConnectorBindingsRevit.Receive.cs

```diff
@@ -333,6 +337,7 @@ void ConvertNestedElements(Base @base, ApplicationObject appObj, bool receiveDir
       if (index % 50 == 0)
       {
         transactionManager.Commit();
+        transactionManager.Start();
```

Comment:
Hey @clairekuang your changes look good. One of the reasons that this issue was happening is because this loop was exiting when index % 50 == 0 (aka it was the 0th object) which was leaving the transaction in a closed state. That was my mistake in originally making this, the transaction should continually be open until we call "Finish" because anyone looking at it would assume that it is open until that point.

It is still good to wrap the object deleting in a subtransaction like you did, but this line is necessary so that this doesn't happen with some other method in the future